src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let price = days * 40;
+
+  if (days >= 7) {
+    price -= 50;
```

Comment:
Replace the literal '7' with a constant to give it a meaningful name and replace '50' with a constant for the long term discount. For example, `const LONG_TERM_DAYS = 7;` and `const LONG_TERM_DISCOUNT = 50;`.